modules/core/src/v2/wallet.ts

```diff
@@ -620,6 +620,16 @@ Wallet.prototype.addresses = function(params, callback) {
   .nodeify(callback);
 };
 
+/**
+ * Gets the keychains queries we will resolve
+ * @param reqId
+ * @returns {*}
+ */
+Wallet.prototype.keychainQueriesForSigning = function(reqId) {
```

Comment:
I'd rather not expose this as part of the public api. This is trivial enough that it should probably just be inline in `signTransaction`

Reply:
ah good point